packages/lesswrong/components/common/SearchBar.tsx

```diff
@@ -83,7 +83,7 @@ const styles = (theme: ThemeType) => ({
     "--icon-size": "24px",
   },
   searchIconButton: {
-    color: isFriendlyUI ? theme.palette.text.bannerAdOverlay : theme.palette.header.text ,
+    color: isFriendlyUI ? theme.palette.grey[600] : theme.palette.header.text ,
```

Comment:
I don't know what find-replace was used for the change that this reverts, but this style edited the wrong side of a forum-gate